src/components/campaignLandingPage/StoriesPanel.module.css

```diff
@@ -0,0 +1,6 @@
+/* 768px = $medium-screen */
+@media (min-width: 768px) {
+  .storyImage {
+    max-width: 40%;
+  }
+}
```

Comment:
This was taken directly from the liquid template (was an inline `<style>` tag inside the template). I didn't find a better way to accomplish this using the standard VA classes. Let me know if there's a better way

Reply:
Was there a media query in the original?